src/pipeline/estimator_pipeline.hpp

```diff
@@ -87,9 +93,17 @@ class CpuEstimatorPipeline : public ResourceEstimatorPipeline {
     if (_visualisation) {
       slackObserver.addConsumer(&slackTimeSeriesExporter);
     }
+
+    // Bus producers
```

Comment:
As far as i know it is only matter of configuration to make it working, but didn't try it yet..